laplace/subnetlaplace.py

```diff
@@ -0,0 +1,129 @@
+import torch
+from torch.distributions import MultivariateNormal
+
+from laplace.baselaplace import FullLaplace
+from laplace.curvature import BackPackGGN
+
+
+__all__ = ['SubnetLaplace']
```

Comment:
I think It's good practice to have this in all files. Of course that's out of scope for this PR but maybe we should make an issue for this? Do you know what exactly this helps with?

Reply:
I think it determines what is imported when you call
```python
from laplace.subnetlaplace import *
```
and maybe there are other effects too.

Reply:
Yes good point -- I added `__all__` for the all the files in `utils/` for now, and will create an issue for any remaining files.